overlord/ifacestate/ifacemgr.go

```diff
@@ -174,6 +174,10 @@ func (m *InterfaceManager) StartUp() error {
 			return err
 		}
 	}
+	if snapdAppArmorServiceIsDisabled() {
+		s.Warnf(`the snapd.apparmor service is disabled; snap applications will likely not start.
```

Comment:
How about issue a similar warning from snap-confine? Looking at the manpage it should set errno to ENOENT when a profile does not exist

Reply:
Good idea, I'll do some test.

Reply:
Yes, ENOENT is returned when the profiles are missing.

I've added a commit with your suggestion.